### DISCUSSION TITLE
osc endpoint to control eye gaze direction on any avatar

### DISCUSSION THREAD

**Discussion by EIA485** [vrchat-community/osc#12] [Ideas]
currently if i want to implement eye tracking via osc id need to do it using parameters. this would require per avatar set up. would love to have some generalized osc endpoints for the eye gaze direction and eye openness.

**Comment by AeroScripts** [comment]
Important to also include something for focus/crosseyed. it's a lot more important for realistic eye tracking than you might think.

Usually this is done by having X axis controlled independently for each eye.
